### ISSUE TITLE
Object descriptions in the documentation

### ISSUE THREAD

**Issue by noobcontrol** [Icinga/icinga2#5567]
In the actual language reference documentation

https://www.icinga.com/docs/icinga2/latest/doc/17-language-reference/#object-definition

it says

'Icinga 2 features an object-based configuration format. You can define new objects using the object keyword:'

I recommend to write

'Icinga 2 features an object-based configuration format. You can define new objects using the object or template keyword:'

and to move the template description right below the object descriptions with the object keyword.

Reason: Objects with the object and template keyword are both objects.

**Comment by Crunsher** [comment]
No, there are no object generated just from a template definition, they are not objects. How did you come to this conclusions?

**Comment by noobcontrol** [comment]
'The default-host and test-host objects are marked as templates using the template keyword. '

https://www.icinga.com/docs/icinga2/latest/doc/17-language-reference/#template-imports

The documentation names templates as objects.

**Comment by dnsmichi** [comment]
templates and objects share the same "namespace", i.e. you cannot have a template called `ping` and an object called `ping`. Still they differ in the way they are used during config compilation. Objects still exist while templates are gone.

**Comment by noobcontrol** [comment]
I don't deny your statements but then the documentation above is a little bit misleading as it denotes templates as objects, too.

**Comment by Crunsher** [comment]
Templates are neither regarded as config objects (dont show up in object list) nor are they runtime objects, as @dnsmichi stated, they are just gone at that point.

So if you have an idea how to reword it without implying or stating there are objects, go ahead ^_^

**Comment by noobcontrol** [comment]
> template Host "default-host" {
>   vars.colour = "red"
> }
> 
> template Host "test-host" {
>   import "default-host"
> 
>   vars.colour = "blue"
> 
>}
> 'The default-host and test-host **objects** are marked as templates using the template keyword. '
https://www.icinga.com/docs/icinga2/latest/doc/17-language-reference/#template-imports

Well you are implying in the documentation that they are objects. Not me. ^^

**Comment by Crunsher** [comment]
That's what I meant, if you want to reword it to _not_ do that you can submit a PR

**Comment by noobcontrol** [comment]
All right. Thanks.
